scripts/index.js

```diff
@@ -26,28 +27,47 @@ const config =
       ? importCwd('./config/webpack.config.dev')
       : importCwd('react-scripts/config/webpack.config.dev');
 
+const spinner = ora('Update webpack configuration').start();
+
+const chromeExtension = Boolean(backgroundScript) && Boolean(contentScript);
+if (chromeExtension) {
+  spinner.info("Chrome extension watching detected")
+  spinner.info(`    backgoundScript[${backgroundScript}]`);
+  spinner.info(`    contentScript[${contentScript}]`);
+}
+
+/**
+ * We need to update the webpack dev config in order to remove the use of webpack devserver
+ */
+config.entry = config.entry.filter(fileName => !fileName.match(/webpackHotDevClient/));
+config.entry = Object.assign(
+  {},
+  {
+    main: [
+      ...config.entry,
+      'webpack-dev-server/client?http://localhost:8080',
```

Comment:
Pretty much like I said on the previous PR:

> I think we'd be better off trying to reuse as much as possible of CRA webpack config. If you look into their webpack config they use a custom webpack dev server client to display better errors to the users.

In that matter we do not need to remove their `webpackHotDevClient` anymore IMO.

Reply:
Hi @Nargonath thanks for getting back to me.

The main problem here is the development set up of a chrome extension, if we use the cra's dev server client, we'll have an exception on this:
https://github.com/facebook/create-react-app/blob/master/packages/react-dev-utils/webpackHotDevClient.js#L60-L69

```
Uncaught SyntaxError: The URL 'chrome-extension:lcfnnoggdagjbblom/sockjs-node' is invalid
    at new SockJS (main.js:70)
    at eval (webpackHotDevClient.js:53)
```

And as far as I know, we can not override the `window.location` value inside the chrome extension.